<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>490</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>490</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>D2Coding ligature</family>
    <pointsize>11</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1301</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>도서 등록 / 수정</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>845</x>
            <y>38</y>
            <width>431</width>
            <height>361</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>  도 서   I D  : </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="input_book_idno">
               <property name="toolTip">
                <string>등록하실 책의 ID Number를 입력하세요.</string>
               </property>
               <property name="toolTipDuration">
                <number>3</number>
               </property>
               <property name="placeholderText">
                <string>등록하실 책의 ID Number를 입력하세요.</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>  분       류  : </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="input_book_type">
               <property name="toolTip">
                <string>등록하실 책의 대분류 번호를 입력하세요.</string>
               </property>
               <property name="toolTipDuration">
                <number>3</number>
               </property>
               <property name="placeholderText">
                <string>등록하실 책의 대분류 번호를 입력하세요.</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>  도 서 제 목  : </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="input_book_name">
               <property name="toolTip">
                <string>등록하실 책의 제목을 입력하세요.</string>
               </property>
               <property name="toolTipDuration">
                <number>3</number>
               </property>
               <property name="placeholderText">
                <string>등록하실 책의 제목을 입력하세요.</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>  저       자  : </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="input_book_date">
               <property name="toolTip">
                <string>등록하실 책의 저자/지은이를 입력하세요.</string>
               </property>
               <property name="toolTipDuration">
                <number>3</number>
               </property>
               <property name="placeholderText">
                <string>등록하실 책의 저자/지은이를 입력하세요.</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>  출 판 일 자  : </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="input_book_edit">
               <property name="toolTip">
                <string>등록하실 책의 출판일자를 입력하세요.</string>
               </property>
               <property name="toolTipDuration">
                <number>3</number>
               </property>
               <property name="placeholderText">
                <string>등록하실 책의 출판일자를 입력하세요.</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>  가       격  : </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="input_book_cost">
               <property name="toolTip">
                <string>등록하실 책의 가격을 입력하세요.</string>
               </property>
               <property name="toolTipDuration">
                <number>3</number>
               </property>
               <property name="placeholderText">
                <string>등록하실 책의 가격을 입력하세요.</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="horizontalLayoutWidget_8">
          <property name="geometry">
           <rect>
            <x>894</x>
            <y>414</y>
            <width>341</width>
            <height>41</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QPushButton" name="btn_add">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>등       록</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_mod">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>수       정</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_del">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>삭       제</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="verticalLayoutWidget_3">
          <property name="geometry">
           <rect>
            <x>14</x>
            <y>26</y>
            <width>801</width>
            <height>411</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QTableWidget" name="tblBooks"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
